src/types/lazy.go

```diff
@@ -329,10 +247,84 @@ func FormatType(s string) (res string) {
 	return "unknown(" + s + ")"
 }
 
-func IsClassType(s string) bool {
-	return strings.HasPrefix(s, `\`) && !IsShapeType(s) && !IsArrayType(s)
+func slice(typ byte, byteFields []uint8, args ...string) []byte {
```

Comment:
Moved to the end of the file so that there are exported functions at the beginning of the file.

Reply:
Revert.